pkg/signature/kms/azure/client.go

```diff
@@ -303,9 +303,27 @@ func (a *azureVaultClient) createKey(ctx context.Context) (crypto.PublicKey, err
 	return a.public()
 }
 
-func (a *azureVaultClient) sign(ctx context.Context, hash []byte) ([]byte, error) {
+func getKeyVaultSignatureAlgo(algo crypto.Hash) (keyvault.JSONWebKeySignatureAlgorithm, error) {
```

Comment:
The code does check when the SignerVerifier is created that it uses a supported algorithm, so this shouldn't ever return a non-nil error. I added it here for completion. I can add a test for this if we want to keep the ability to return an error.